.github/workflows/test_suite_linux.yml

```diff
@@ -102,8 +102,6 @@ jobs:
         path: artifact
     - name: Extract Webots
       run: tar xjf artifact/webots-*-x86-64*.tar.bz2 -C artifact
-    - name: Install Webots Dependencies
-      run: sudo scripts/install/linux_test_dependencies.sh
```

Comment:
This script installs runtime dependencies as well. We need an argument (e.g. `--exclude-ros`) to exclude only the ROS dependency.